build.fsx

```diff
@@ -61,7 +62,7 @@ let gitName = "Paket"
 // The url for the raw files hosted
 let gitRaw = environVarOrDefault "gitRaw" "https://raw.github.com/fsprojects"
 
-let dotnetcliVersion = "1.0.0-preview4-004233"
+let dotnetcliVersion = "2.0.0-alpha-005165"
```

Comment:
plus I used the latest so I could be sure the slimmed down fsproj and csproj would be compatible